### PR TITLE
Bug fixes for Secure Video, Wi-Fi Router & permissions

### PR DESCRIPTION
**Pull request by codyc1515**

### CODE REVIEW

File: src/lib/gen/HomeKit-TV.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class ActiveIdentifier extends Characteristic {
     super('Active Identifier', ActiveIdentifier.UUID);
     this.setProps({
       format: Formats.UINT32,
-      perms: [Perms.READ, Perms.WRITE, Perms.NOTIFY]
```
Comment (KhaosT) [inline src/lib/gen/HomeKit-TV.ts]: 👋 It really needs the write permission...

**Comment by codyc1515** [comment]
Have reverted this. I think I made the same mistake last time. I don't know why the spec doesn't reflect this.

**Comment by Supereg** [comment]
Out of interest, what's the ProductData characteristic for?

**Comment by Supereg** [comment]
Additionally, my resources and informations suggest different values for most changes you suggested. Could you specify where you got your information from?

**Comment by KhaosT** [comment]
@Supereg Those matched Logi Circle 2's implementation I think? https://github.com/KhaosT/HAP-NodeJS/issues/741

**Pull request merged**